.typos.toml

```diff
@@ -165,13 +165,18 @@ combintation = "combintation"
 # true positives
 [type.rust]
 extend-ignore-re = [
+  # src/generated/cloud/dialogflow/v2 has some typos, ignoring "whe" everywhere
+  # seems too broad.
+  " whe ",
```

Comment:
remove, appears below

Reply:
Good catch, thanks. Removed the one below to keep the alphabetical-ish.